components/timezone/src/ixdtf.rs

```diff
@@ -36,6 +34,8 @@ pub enum ParseError {
     InvalidOffsetError,
     /// There was an invalid IANA identifier provided.
     InvalidIanaIdentifier,
```

Comment:
I don't think we ever want to raise `InvalidIanaIdentifier`, it might already be dead

Reply:
Done